src/engine/source/server/src/endpoints/unixDatagram.cpp

```diff
@@ -0,0 +1,264 @@
+#include <server/endpoints/unixDatagram.hpp>
+
+#include <cstring>      // Unix  socket datagram bind
+#include <fcntl.h>      // Unix socket datagram bind
+#include <sys/socket.h> // Unix socket datagram bind
+#include <sys/un.h>     // Unix socket datagram bind
+#include <unistd.h>     // Unix socket datagram bind
+
+#include <logging/logging.hpp>
+#include <uvw.hpp>
+
+namespace
+{
+constexpr unsigned int MAX_MSG_SIZE {65536 + 512}; ///< Maximum message size (TODO: I think this should be 65507)
+} // namespace
+
+namespace engineserver::endpoint
+{
+UnixDatagram::UnixDatagram(const std::string& address,
+                           const std::function<void(std::string&&)>& callback,
+                           const std::size_t taskQueueSize)
+    : Endpoint(address, taskQueueSize)
+    , m_callback(callback)
+    , m_handle(nullptr)
+    , m_bufferSize(-1)
+{
+    if (address.empty())
+    {
+        throw std::runtime_error("Address must not be empty");
+    }
+
+    if (address.length() >= sizeof(sockaddr_un::sun_path))
+    {
+        auto msg = fmt::format("Path '{}' too long, maximum length is {} ", address, sizeof(sockaddr_un::sun_path));
+        throw std::runtime_error(msg);
```

Comment:
Can we directly use the fmt::format? The same for every occurrence

Reply:
No, it is not necessary.